recipe/meta.yaml

```
@@ -55,8 +56,8 @@ requirements:
   build:
      # ensure that all variants of cuda_compiler_version are tested
     - {{ compiler('cuda') }}              # [cuda_compiler_version != "None"]
-    - {{ compiler('c') }}                 # [cuda_compiler_version != "None"]
-    - {{ stdlib('c') }}                   # [cuda_compiler_version != "None"]
+    - {{ compiler('c') }}                 # [cuda_compiler_version != "None" or build_platform != target_platform]
+    - {{ stdlib('c') }}                   # [cuda_compiler_version != "None" or build_platform != target_platform]
     - python                              # [build_platform != target_platform]
     - cross-python_{{ target_platform }}  # [build_platform != target_platform]
```

Maybe we should only use `cross-python` when the other compilers are used (so CUDA only)?

```suggestion
    - {{ compiler('c') }}                 # [cuda_compiler_version != "None"]
    - {{ stdlib('c') }}                   # [cuda_compiler_version != "None"]
    - python                              # [cuda_compiler_version != "None" and build_platform != target_platform]
    - cross-python_{{ target_platform }}  # [cuda_compiler_version != "None" and build_platform != target_platform]
```